Source/CesiumRuntime/Private/CesiumTile.cpp

```diff
@@ -0,0 +1,34 @@
+#include "CesiumTile.h"
+#include "CalcBounds.h"
+#include "CesiumTransforms.h"
+#include "Components/PrimitiveComponent.h"
+#include "VecMath.h"
+
+bool UCesiumTile::TileBoundsOverlapsPrimitive(
+    const UPrimitiveComponent* Other) const {
+  if (IsValid(Other)) {
+    return Bounds.GetBox().Intersect(Other->Bounds.GetBox()) &&
+           Bounds.GetSphere().Intersects(Other->Bounds.GetSphere());
+  } else {
+    return true;
```

Comment:
It's more intuitive to me if this returns false, because a tile can't overlap with something that doesn't exist anymore.